geomesa-gs-plugin/geomesa-accumulo-gs-plugin/README.md

```diff
@@ -1,11 +1,33 @@
-# GeoMesa Plugin
+# GeoMesa Accumulo GeoServer Plugin
```

Comment:
I think this should mention the geoserver version that is being targeted since it may differ or is likely to change